lib/takeaway.rb

```diff
@@ -0,0 +1,64 @@
+require 'twilio-ruby'
+require_relative 'menu'
+require_relative 'sms'
+
+class Takeaway
+
+
+  attr_reader :menu, :order
+
+  def initialize(menu = Menu.new)
+    @menu = menu
+    @order = {}
+#    @sms = sms
+  end
+
+  def view_menu
+    menu.print_menu
+  end
+
+
+  # need to think about what place order will do at the moment it is responsible for sending the text!!!
+  def place_order
```

Comment:
I actually think this method is probably ok.  If we wanted to develop the application further, we'd probably want to implement a way of sending on the completed order to the restaurant or something similar, but, for now, the spec only tells us to send a confirmation to the user when an order is placed which is exactly what you're doing.